be/src/storage/CMakeLists.txt

```diff
@@ -249,15 +249,17 @@ add_library(Storage STATIC
     rows_mapper.cpp
     primary_key_compaction_conflict_resolver.cpp
     local_primary_key_compaction_conflict_resolver.cpp
-    inverted/index_descriptor.hpp
-    inverted/inverted_index_iterator.cpp
-    inverted/inverted_index_iterator.cpp
-    inverted/inverted_index_option.cpp
-    inverted/inverted_index_common.hpp
-    inverted/inverted_plugin_factory.cpp
-    inverted/clucene/clucene_plugin.cpp
-    inverted/clucene/clucene_roaring_hit_collector.hpp
-    inverted/clucene/clucene_inverted_writer.cpp
-    inverted/clucene/clucene_inverted_reader.cpp
-    inverted/clucene/clucene_inverted_util.hpp
-    inverted/clucene/match_operator.cpp)
+    index/index_descriptor.hpp
+    index/inverted/inverted_index_iterator.cpp
+    index/inverted/inverted_index_option.cpp
+    index/inverted/inverted_index_common.hpp
```

Comment:
specific reason of using `.hpp`?

Reply:
No, I suppose it was just a naming mistake, I fixed it.